### ISSUE TITLE
Logstash

### ISSUE THREAD

**Issue by caevv** [cloudposse/terraform-aws-elasticsearch#59]
> Terraform module to provision an Elasticsearch cluster with built-in integrations with Kibana and Logstash.

Apologies if is somewhere obvious, but where is the logstash, the host and how to configure it? I can't see anything about logstash here.

Thanks

**Comment by Gowiem** [comment]
@caevv good point. I think our repo is poorly worded. @osterman / @cloudposse/admins can one of ya'll remove the bit about logstash from this repository's description? It's a bit misleading and not necessary as this isn't a `terraform-aws-ELK` module.
